### PR TITLE
Radio Input Fixes

### PR DESCRIPTION
**Pull request by PiyushPawar17**
Clubhouse Story Link: https://app.clubhouse.io/clarisights/story/20421/knit-ui-radio-input-fixes
Clubhouse Story ID: [ch20421]

### CODE REVIEW

File: src/components/Radio/styledRadio.tsx [hunk 1]
```diff
@@ -33,9 +33,14 @@ export const StyledRadioRoot = styled.span<RadioWrapperProps>`
 
     &.knit-radio-checked {
       .knit-radio-inner {
-        &:after {
-          transform: scale(1);
-          -webkit-transform: scale(1);
+        position: relative;
+
+        &::after {
+          top: 50%;
```
Comment (anshumanv) [inline src/components/Radio/styledRadio.tsx]: Move these to the `StyledRadioInner ` SC

File: src/components/Radio/styledRadio.tsx [hunk 2]
```diff
@@ -33,9 +33,14 @@ export const StyledRadioRoot = styled.span<RadioWrapperProps>`
 
     &.knit-radio-checked {
       .knit-radio-inner {
-        &:after {
```
Comment (anshumanv) [inline src/components/Radio/styledRadio.tsx]: `&:` also works but `&::` is better which clearly says that we are working with pseudo-elements, nice

File: src/components/Radio/styledRadio.tsx [hunk 3]
```diff
@@ -33,16 +33,15 @@ export const StyledRadioRoot = styled.span<RadioWrapperProps>`
...
-          transform: scale(1);
-          -webkit-transform: scale(1);
+        position: relative;
+
+        &::after {
+          top: 50%;
+          left: 50%;
```
Comment (anshumanv) [inline src/components/Radio/styledRadio.tsx]: You can remove these from here

File: src/components/Radio/styledRadio.tsx [hunk 4]
```diff
@@ -77,21 +68,24 @@ export const StyledRadioInner = styled.span<{
...
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    -webkit-transform: translate(-50%, -50%);
+    -ms-transform: translate(-50%, -50%);
     border-radius: 0.7rem;
     border-top: 0;
     border-left: 0;
     content: " ";
     background-color: ${Blue100.hex};
```
Comment (prasook-jain) [inline src/components/Radio/styledRadio.tsx]: Please use color using `getThemeColor` util function..

File: src/components/Radio/styledRadio.tsx [hunk 5]
```diff
@@ -77,21 +68,24 @@ export const StyledRadioInner = styled.span<{
...
+  &::after {
     position: absolute;
     width: ${({ theme, size }) => getRadioBeadSize(theme, size)};
     height: ${({ theme, size }) => getRadioBeadSize(theme, size)};
-    left: 2.5px;
-    top: 2.5px;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    -webkit-transform: translate(-50%, -50%);
```
Comment (prasook-jain) [inline src/components/Radio/styledRadio.tsx]: we don't need do prefixing.. it's automatically done by styled-component.

**Pull request merged**